tests/unit/mergeHeadContents.test.ts

```diff
@@ -0,0 +1,80 @@
+// FILEPATH: /Users/philipp/Projects/@swup/head-plugin/tests/unit/mergeHeadContents.test.ts
```

Comment:
What's this comment doing?

Reply:
Nothing, of course 🥴 Looks automatically generated, so I assumed it had to do with some obscure vitest feature, but turns it it's entirely optional.